src/future/mod.rs

```diff
@@ -51,6 +51,7 @@ pub use async_macros::{select, try_select};
 use cfg_if::cfg_if;
 
 pub use future::Future;
+pub use future::FutureExt;
```

Comment:
We shouldn't export the `FutureExt` trait directly here; instead it should be part of the `prelude`, similar to `StreamExt`.